### ISSUE TITLE
wrong box ordering

### ISSUE THREAD

**Issue by DdHms** [qqwweee/keras-yolo3#602]
Hi,
I'm pretty sure there is a mistake in 

> yolo.py ln 142

`
top, left, bottom, right = box
`
The order should be: 
`left, top, right, bottom = box
`
according to the convention and annotations order: 'xmin, ymin, xmax, ymax'

**Comment by tabmoo** [comment]
Just try changing it and see. The code is correct.

**Comment by DdHms** [comment]
I did change it. After the change it works beforehand I get inverted boxes

**Comment by tabmoo** [comment]
Hhm. In my case it is the opposite...
Please, if you are interested in bugs in this repo, look at this issue with the loss function: https://github.com/AlexeyAB/darknet/issues/4594.

**Comment by tabmoo** [comment]
I think this ordering comes from yolo_correct_boxes where x any y are reversed.

**Comment by DdHms** [comment]
Yes, quite confusing if you try to evaluate predicted vs annotated or draw both on the same image
